tests/lax_numpy_test.py

```diff
@@ -4670,6 +4670,41 @@ def testMgrid(self):
                         atol=atol,
                         rtol=rtol)
 
+  def testOgrid(self):
+    def assertListOfArraysEqual(xs, ys):
+      self.assertTrue(isinstance(xs, list))
+      self.assertTrue(isinstance(ys, list))
```

Comment:
I think we actually have `assertIsInstance`, which will give more informative error messages on failure

Reply:
Yes that's a better way :)